### ISSUE TITLE
[BUG] frozendict.deepfreeze is not in __init__.pyi

### ISSUE THREAD

**Issue by jclerman** [Marco-Sulla/python-frozendict#99]
My IDE complains that `frozendict.deepfreeze()` is not advertised in `__init__.pyi`.

* OS version: MacOS 13.6.6
* Python3 version (python3 -V -V): 
`Python 3.10.12 (main, Aug 16 2023, 12:57:33) [Clang 14.0.0 (clang-1400.0.29.202)]`

**Steps to reproduce:**

1. In PyCharm, write code that accesses the deepfreeze function, like:
```
import deepfreeze

def temp(x):
    return frozendict.deepfreeze(x)
```

**Actual result:**
I get a warning flagged in PyCharm (even though the code runs fine):

![image](https://github.com/Marco-Sulla/python-frozendict/assets/19315670/5b817820-0689-4359-a20e-b12c87122429)

**Comment by jclerman** [comment]
NOTE: This also causes `mypy` to complain thusly:

```
error: Module has no attribute "deepfreeze"  [attr-defined]
```

**Comment by Marco-Sulla** [comment]
ty for the issue. Fixed in 2.4.3

**Comment by jclerman** [comment]
Thanks for the quick fix!!

By the way, when looking into how this was done I happened upon this notice from GitHub. Not sure if it's worth fixing - looks like maybe the tag was applied to a deleted branch instead of to the main branch?

![image](https://github.com/Marco-Sulla/python-frozendict/assets/19315670/572f74f7-2775-4c88-b787-635876c8106f)

**Comment by Marco-Sulla** [comment]
I have done some mess with the push. I don't know what happened in detail. Anyway, it seems that 2.4.3 contains only the fix of enum with deepfreeze. I just released 2.4.4, that now contains the fix for this issue.
